### PR TITLE
feat: add `init_standalone` method to Pallet class

### PR DESCRIPTION
**Pull request by stdavis**
This makes running pallets outside of forklift easier.

### CODE REVIEW

**Comment by stdavis** [comment]
Actually, let me add a test for this method...

**Comment by stdavis** [comment]
Nevermind. I couldn't come up with a meaningful test. This is ready for a review...

File: src/forklift/models.py [hunk 1]
```diff
@@ -233,6 +233,16 @@ def add_packing_slip(self, slip):
         for slip_crate in slip['crates']:
             crate_objects_by_name[slip_crate['name']].result = [slip_crate['result']]
 
+    def init_standalone(self):
```
Comment (steveoh) [inline src/forklift/models.py]:
```suggestion
    def configure_standalone_logging(self):
```

File: samples/RunablePallet.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    def lift(self):
+        self.log.info('lifting pallet')
+
+    def ship(self):
+        self.log.info('shipping pallet')
+
+
+if __name__ == '__main__':
+    pallet = RunablePallet()
+    pallet.init_standalone()
```
Comment (steveoh) [inline samples/RunablePallet.py]:
```suggestion
    pallet.configure_standalone_logging()
```

File: src/forklift/models.py (hunk 1 above)
Comment (stdavis) [reply src/forklift/models.py]: My thought was that eventually, it may have more than just logging in it. But I'm good with whatever.

**Comment by steveoh** [comment]
Should we doc this in the readme somewhere?

**Comment by stdavis** [comment]
I don't see a reason to. We don't have any documentation on the Pallet class there. It's all in the code.

**Pull request merged**